README.md

```diff
@@ -72,3 +72,7 @@ by using the `service-b.service.consul` name.
 - `service-b` is not declaring any `@PortBinding` because our test does not need to talk to it.
 Only `service-a` will talk to it, but it can do it by using the container IP to which
 the `service-b.service.consul` name resolves to.
+
+---
+By default, the `dockerunit-consul` module will use the DockerHub Consul image. If you want to use a different image as
+Consul image, you need to pass the name of the new image using the System property called `consul.image`.
```

Comment:
wdyt to not make it personal with

By default the `dockerunit-consul` module uses Consul image from DockerHub. To override the default using a different 
Consul Docker image the System property `consul.image` can be provided.